boards/shields/max3421e/Kconfig.defconfig

```diff
@@ -1,9 +0,0 @@
-# Copyright (c) 2022 Nordic Semiconductor ASA
-# SPDX-License-Identifier: Apache-2.0
-
-if SHIELD_SPARKFUN_MAX3421E
-
-config UHC_DRIVER
```

Comment:
@jfischer-no I'd like you to verify on this specific change.
To me, there is no reason this driver doesn't follow the same rule as others

Reply:
Change is correct, thanks.